src/status_im/ui/components/emoji_thumbnail/color_picker.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im.ui.components.emoji-thumbnail.color-picker
+  (:require [quo.react-native :as rn]
+            [status-im.ui.components.emoji-thumbnail.styles :as styles]))
+
+(defn colors-row1 [color-circle]
+  [rn/view {:style (merge styles/emoji-picker-color-row-container styles/emoji-picker-row1-style)}
+   (for [x styles/emoji-picker-colors-row1]
+     [color-circle x])])
+
+(defn colors-row2 [color-circle]
+  [rn/view {:style (merge styles/emoji-picker-color-row-container styles/emoji-picker-row2-style)}
+   (for [x styles/emoji-picker-colors-row2]
+     [color-circle x])])
+
+(defn colors-row3 [color-circle]
+  [rn/view {:style (merge styles/emoji-picker-color-row-container styles/emoji-picker-row3-style)}
+   (for [x styles/emoji-picker-colors-row3]
+     [color-circle x])])
+
+(defn color-picker-section [color-circle]
+  [:<>
+   [colors-row1 color-circle]
```

Comment:
you could have only one component and pass styles as parameters

Reply:
Yes, that's great advice, will do that 👍, Thanks